docs/tutorials/scheduling.rst

```diff
@@ -0,0 +1,258 @@
+schedule the targets in.
+
+First we define our `~astroplan.Observer` object (where we are observing from):
+
+.. code-block:: python
+
+    >>> from astroplan import Observer
+
+    >>> apo = Observer.at_site('apo')
+
+Now we want to define our list of targets (`~astroplan.FixedTarget` objects),
+any object that is in SIMBAD can be called by an identifier.
+
+.. code-block:: python
+
+    >>> from astroplan import FixedTarget
+
+    >>> Deneb = FixedTarget.from_name('deneb')
+    >>> M13 = FixedTarget.from_name('m13')
+
+    >>> Deneb
+    <FixedTarget "deneb" at SkyCoord (ICRS): (ra, dec) in deg (310.35797975, 45.28033881)>
+
+    >>> M13
+    <FixedTarget "m13" at SkyCoord (ICRS): (ra, dec) in deg (250.423475, 36.4613194)>
+
+We also need to define when we will be observing the targets, `~astropy.time.Time`
+objects for the start and end of our observing window. The half night goes from 7PM
+local time to 1AM local time, in UTC this will be from 2AM to 8AM.
+
+.. code-block:: python
+
+    >>> from astropy.time import Time
+
+    >>> start_time = Time('2016-06-07 02:00')
+    >>> end_time = Time('2016-06-07 08:00')
+
+:ref:`Return to Top <scheduling_tutorial>`
+
+.. _scheduling-creating_constraints_and_observing_blocks:
+
+Creating Constraints and Observing Blocks
+=========================================
+
+An in-depth tutorial on creating and using constraints can be found in
+the :ref:`constraint tutorial <constraints>`.
+
+Constraints, when evaluated, take targets and times, and give scores that
+indicate how well the combination of target and time fulfill the constraint.
+We want to make sure that our targets will be high in the sky while observed
+and that they will be observed during the night. We don't want any object to
+be observed at an airmass greater than 3, but we prefer a better airmass.
+Usually constraints scores are boolean, but with ``boolean_constraint = False``
+the constraint will output floats instead, indicated when it is closer to ideal.
+
+.. code-block:: python
+
+    >>> from astroplan.constraints import AtNightConstraint, AirmassConstraint
+
+    >>> global_constraints = [AirmassConstraint(max = 3, boolean_constraint = False),
+    ...                       AtNightConstraint()]
+
+Now that we have constraints that we will apply to every target, we need to
+create an   `~astroplan.ObservingBlock` for each target+configuration
+combination. An observing block needs a target, a duration, and a priority;
+configuration information can also be given (i.e. filter, instrument, etc.).
+For each filter we want 17 exposures per target (100 seconds for M13 and 60
+seconds for Deneb) and the instrument has a read-out time of 20 seconds.
+We also need to consider that the moon can be a problem for observations
+in the B filter, so we want the moon to be down or dim for the B filter.
+
+.. code-block:: python
+
+    >>> from astroplan import ObservingBlock
```

Comment:
If you're defining a code block that won't produce any printed outputs, don't use `>>>`.